kore/test/Test/Kore/Repl/Interpreter.hs

```diff
@@ -229,13 +230,28 @@ unificationFailure =
         claim = zeroToTen
         command = Try . Left $ AxiomIndex 0
     in do
-        Result { output, continue, state } <- run command axioms claim
+        Result { output, continue, state } <- run command axioms [claim] claim
         expectedOutput <-
             unificationErrorMessage cannotUnifyDistinctDomainValues one zero
         output `equalsOutput` expectedOutput
         continue `equals` True
         state `hasCurrentNode` ReplNode 0
 
+proofStatus :: IO ()
+proofStatus =
+    let
+        claims = [zeroToTen, emptyClaim]
+        claim = zeroToTen
+        axioms = [add1]
+        command = ProofStatus
+    in do
+        Result { output, continue } <-
+            run command axioms claims claim
+        output `equalsOutput` "Current proof status: \n  \
```

Comment:
Generally, I would prefer opting for exporting the function that formats the data type into a string and use that instead of typing strings in tests. Would that be doable here?